core/src/main/scala/reftree/render/ImageRenderer.scala

```diff
@@ -7,11 +7,12 @@ import uk.co.turingatemyhamster.graphvizs.exec._
 
 import scala.sys.process.{Process, BasicIO}
 
-object PngRenderer {
-  def renderPng(graph: Graph, output: Path, options: RenderingOptions): Unit = {
+object ImageRenderer {
```

Comment:
Some of the formats supported by `dot` are not images (e.g. `dot`, `plain`). Maybe `DotRenderer`?